src/Pim/Bundle/CatalogBundle/Manager/MediaManager.php

```
@@ -139,7 +139,11 @@ public function handle(ProductMediaInterface $media, $filenamePrefix)
      */
     public function duplicate(ProductMediaInterface $source, ProductMediaInterface $target, $filenamePrefix)
     {
-        $target->setFile(new File($source->getFilePath()));
+        if (null === $path = $this->getFilePath($source)) {
+            return;
```

Hmm, we just silently do nothing if the source media is not valid? Maybe an exception?